src/com/github/kornilova203/matlab/completion/MatlabKeywordCompletionContributor.kt

```diff
@@ -35,6 +43,10 @@ class MatlabKeywordCompletionContributor : CompletionContributor() {
     private fun provider(vararg keywords: String): CompletionProvider<CompletionParameters> {
         return object : CompletionProvider<CompletionParameters>() {
             override fun addCompletions(parameters: CompletionParameters, context: ProcessingContext, result: CompletionResultSet) {
+                val position = parameters.position
+                if (AFTER_INTEGER_LITERAL.accepts(position) || AFTER_FLOAT_LITERAL.accepts(position)) {
+                    return
```

Comment:
Please add `result.stopHere()` before `return`.
It will disable completion of identifiers from current file